src/common/res/features/l10n/settings.json

```diff
@@ -14,129 +14,139 @@
             "value": "1"
         }, 
         {
-            "name": "Czech (100%)", 
+            "name": "Chinese Simplified (3%)", 
             "value": "2"
         }, 
         {
-            "name": "Danish (100%)", 
+            "name": "Czech (100%)", 
             "value": "3"
```

Comment:
@egens, looks like we'll need to use the name of the language before we pull this in, or exclude Chinese Simplified temporarily.